src/alphabase/base64url.cljc

```diff
@@ -0,0 +1,28 @@
+(ns alphabase.base64url
+  "Base 64 optimized namespace"
+  (:require [clojure.set :as set]
+            #?@(:cljs [[goog.crypt.base64 :as g-base64]]))
+  #?(:clj (:import (java.util Base64))))
+
+(def ^:const base64url-chars "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.")
+(def ^:const base64url-set (set base64url-chars))
+
+(defn encode
+  "Converts a byte array into a base65URL string."
+  ^String
+  [ba]
+  #?(:clj (-> (Base64/getUrlEncoder)
+              (.encodeToString ba))
```

Comment:
this is so minor it probably doesn't matter, but i think executing this function and `decode` below will result in some reflection on the url encoder object.

Reply:
Good thought! I set *warn-on-reflection* true in the `core` ns and fixed all the warnings - it didn't flag this but it did find several others.